osprepare/distro.go

```diff
@@ -231,14 +231,14 @@ func sudoFormatCommand(command string, packages []string) bool {
 	c.Stderr = os.Stderr
```

Comment:
The actually output from apt-get and the other installers is still going to stdout/stderr which isn't really very useful except on a developer's machine.  I think we probably want to log this information, so could you also redirect this to the logs are well?